.vscode/launch.json

```diff
@@ -38,14 +38,17 @@
             "preLaunchTask": "Build and Deploy b-l475e-iot01a"
         },
         {
-            "name": "NXP: Local J-Link",
+            "name": "NXP mimxrt1060: Local J-Link",
             "type": "cortex-debug",
             "request": "launch",
             "servertype":"jlink",
-            "runToMain": true,
+            "postStartSessionCommands": ["monitor reset"],
+            //"runToMain": true,
```

Comment:
should this be commented out or removed?

Reply:
currently it fails to debug when using this flag to true. I have added comment for this.